src/routes/safes/safes.controller.overview.spec.ts

```diff
@@ -1036,22 +1036,181 @@ describe('Safes Controller Overview (Unit)', () => {
   });
 
   describe('Validation', () => {
-    it('forwards error responses from the Config Service', async () => {
-      const chain = chainBuilder().with('chainId', '10').build();
-      const safeInfo = safeBuilder().build();
+    it('omits erroroneous Safes on chains returning an error from the Config Service', async () => {
```

Comment:
Nit: `erroneous`

Reply:
Fixed in 56fee09f357b6eab83f577eebe3fd43ed21db7fb.